From d79d7f9b41ae91bec6c7d5bf80697e0948a46dc2 Mon Sep 17 00:00:00 2001
From: Yongsheng Chen <yongsheng.chen@intel.com>
Date: Fri, 12 Oct 2018 10:30:53 -0700
Subject: [PATCH 2267/2278] media: intel-ipu4: ox03a10: Use the correct LLP
 value

Message for Open Source:
Change the LLP value from 2168 to 2700.

Message for Internal:
Change the LLP value from 2168 to 2700.
The correct LLP calculation should be:
LLP = Horizontal_timing_length_DCG +
      Horizontal_timing_length_VS

Change-Id: I435b4d04684dffe9a145d03c761e5b92801bf037
Signed-off-by: Chen, Yongsheng <yongsheng.chen@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/i2c/crlmodule/crl_ox03a10_configuration.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/i2c/crlmodule/crl_ox03a10_configuration.h b/drivers/media/i2c/crlmodule/crl_ox03a10_configuration.h
index 4f4a853..fa62fad 100644
--- a/drivers/media/i2c/crlmodule/crl_ox03a10_configuration.h
+++ b/drivers/media/i2c/crlmodule/crl_ox03a10_configuration.h
@@ -1460,7 +1460,7 @@ struct crl_v4l2_ctrl ox03a10_v4l2_ctrls[] = {
 		.data.std_data.min = 1920,
 		.data.std_data.max = 65535,
 		.data.std_data.step = 1,
-		.data.std_data.def = 2168,
+		.data.std_data.def = 2700,
 		.flags = V4L2_CTRL_FLAG_UPDATE,
 		.impact = CRL_IMPACTS_NO_IMPACT,
 		.ctrl = 0,
-- 
2.7.4

